modules/navier_stokes/doc/content/source/functormaterials/INSFVkEpsilonViscosityMaterial.md

```diff
@@ -13,8 +13,8 @@ where:
 - $k$ is the turbulent kinetic energy,
 - $\epsilon$ is the turbulent kinetic energy dissipation rate.
 
-!syntax parameters /Materials/INSFVkEpsilonViscosityMaterial
+!syntax parameters /FunctorMaterials/INSFVkEpsilonViscosityMaterial
 
-!syntax inputs /Materials/INSFVkEpsilonViscosityMaterial
+!syntax inputs /FunctorMaterials/INSFVkEpsilonViscosityMaterial
 
-!syntax children /Materials/INSFVkEpsilonViscosityMaterial
+!syntax children /FunctorMaterials/INSFVkEpsilonViscosityMaterial
```

Comment:
To be consistent with your other changes seems like this should now have the suffix `FunctorMaterial`

Reply:
will do

Reply:
done